### ISSUE TITLE
Custom websub listener experiences timeout error intermittently.

### ISSUE THREAD

**Issue by MadhurangaWije** [ballerina-platform/ballerina-standard-library#1422]
**Description:**
Custom websub listener developed in the [Ballerina GitHub connector](https://github.com/MadhurangaWije/module-ballerinax-github/tree/bugfixes/pagination/github/modules/webhook), experiences an intermittent timeout for event payloads dispatched from the GitHub. 

`ballerina-internal.log` file shows the following log when this issue takes place.
```
[2021-06-04 08:27:51,690] SEVERE {b7a.log.crash} - No such error: ServiceExecutionError 
error: No such error: ServiceExecutionError 
```

Following is the error shown in the GitHub webhook dashboard
![Screenshot from 2021-06-04 09-00-23](https://user-images.githubusercontent.com/27579184/120741887-8fa5f280-c513-11eb-8545-a2bd8f64c046.png)

This issue was not experienced prior to SL-Alpha5, and the connector did not undergo any changes during the migration to Alpha5.

**Affected Versions:**
Ballerina Swan Lake Alpha 5

**OS, DB, other environment details and versions:**
OpenJDK 11, Ubuntu 20.04

**Suggested Labels (optional):**
priority/highest

**Comment by ayeshLK** [comment]
* We replayed the failed payloads against the current `websub` framework and was unable to reproduce the issue there.
* Hence we suspect that the original issue is with the Ballerina GitHub connector.

**Comment by MadhurangaWije** [comment]
Yes, we identified an issue in the GitHub connector, and it was fixed in [PR](https://github.com/ballerina-platform/module-ballerinax-github/pull/118). From the websub module, it ll be great if we can get detail error log in place of the following  log in the `ballerina-internal.log`.

The internal error has been casting null to string in the following code block.

```
string eventAction = <string> content["action"];    
```

```
[2021-06-04 08:27:51,690] SEVERE {b7a.log.crash} - No such error: ServiceExecutionError 
error: No such error: ServiceExecutionError 
```
